src/Runner/MediaInfoCommandRunner.php

```diff
@@ -63,10 +62,10 @@ public function __construct($filePath, array $arguments = null, $processBuilder
      */
     public function run()
     {
-        $this->processBuilder->add($this->filePath);
+        $lc_ctype = setlocale(LC_CTYPE, 0);
+        $this->processBuilder->add($this->filePath)->addEnvironmentVariables(array('LANG' => $lc_ctype));
```

Comment:
Yes 


Atenciosamente, 




	

Alan Gregory 

Opcão Telecom 

Departamento de Redes 

+55 45 3253-2277 | 45 9907-9669 

alan @opcaonet.com.br 

skype: alan@opcaonet.com.br 



De: "Maxime Horcholle" <notifications@github.com> 
Para: "mhor/php-mediainfo" <php-mediainfo@noreply.github.com> 
Cc: "alan" <alan@opcaonet.com.br> 
Enviadas: Segunda-feira, 21 de dezembro de 2015 14:19:58 
Assunto: Re: [php-mediainfo] Set LANG env variable (#43) 



In src/Runner/MediaInfoCommandRunner.php : 
> @@ -63,10 +62,10 @@ public function __construct($filePath, array $arguments = null, $processBuilder
>       */
>      public function run()
>      {
> -        $this->processBuilder->add($this->filePath);
> +        $lc_ctype = setlocale(LC_CTYPE, 0);
> +        $this->processBuilder->add($this->filePath)->addEnvironmentVariables(array('LANG' => $lc_ctype)); 


@alangregory could you replace addEnvironmentVariables(array('LANG' => $lc_ctype) by setEnv('LANG', $lc_type) https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Process/ProcessBuilder.php#L142 

— 
Reply to this email directly or view it on GitHub .